### PR TITLE
[Related #105] Race page refinements

### PR DESCRIPTION
**Pull request by supunTE**

### CODE REVIEW

File: apps/client/src/router.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import { NotFound } from './pages/NotFound';
+import { GuardedRoute } from './utils/guardedRoute';
+import { Home, Layout, Leaderboard, Race, Room } from './pages';
+
+export function Router(): ReactElement {
```
Comment (dalanad) [inline apps/client/src/router.tsx:8]: Shall we go for a name like `AppRouter` ?

File: apps/client/src/pages/race/templates/race-text/RaceText.template.tsx [hunk 2]
```diff
@@ -295,16 +319,32 @@ export function RaceText({
                     indexConverter.isCursorAtChar(charIndex, {
                       cursorsAt: otherPlayerCursors,
                     });
+                  /* Show normal cursor if no invalid chars */
+                  const isVisibleRegularCursor =
```
Comment (dalanad) [inline apps/client/src/pages/race/templates/race-text/RaceText.template.tsx:323]: these names should ne `isRegularCursor visible` &`isInvalidCursorVisible`

File: apps/client/src/pages/race/templates/race-text/RaceText.template.tsx [hunk 3]
```diff
@@ -280,6 +300,10 @@ export function RaceText({
                     charIndex,
                     { cursorAt: playerCursorAt },
                   );
+                  const isCursorAtInvalidCursor = indexConverter.isCursorAtChar(
```
Comment (dalanad) [inline apps/client/src/pages/race/templates/race-text/RaceText.template.tsx:303]: this has Cursor twise in it's name  should be `isCursorAtInvalidCharacter` right ?

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/client/src/pages/race/templates/race-text/RaceText.template.tsx (hunk 2 above)
Comment (supunTE) [reply apps/client/src/pages/race/templates/race-text/RaceText.template.tsx:323]: renamed in #114 a2f934c4d69cdfed5c031f3d9174c8e4a76330e6

File: apps/client/src/pages/race/templates/race-text/RaceText.template.tsx (hunk 3 above)
Comment (supunTE) [reply apps/client/src/pages/race/templates/race-text/RaceText.template.tsx:303]: renamed in #114 a2f934c4d69cdfed5c031f3d9174c8e4a76330e6

File: apps/client/src/router.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/router.tsx:8]: renamed this in #114 ab14c6e1287fc13ba3a92c9facc56607020b84ff

**Review by dalanad — APPROVED** [review]

**Pull request merged**